packages/marshal/src/encodePassable.js

```diff
@@ -346,7 +567,7 @@ export const makeEncodePassable = (encodeOptions = {}) => {
         return encodeBinary64(passable);
       }
       case 'string': {
-        return `s${passable}`;
+        return `s${encodeStringSuffix(passable)}`;
       }
       case 'boolean': {
         return `b${passable}`;
```

Comment:
While we're making changes, should we also replace "btrue" and "bfalse" with e.g. "bt" and "bf"?

Reply:
`b1` and `b0`?